aksel.nav.no/website/components/token-docs/token/example/ColorToken.tsx

```diff
@@ -0,0 +1,59 @@
+import { PaletteIcon } from "@navikt/aksel-icons";
+import { Box, Heading, VStack } from "@navikt/ds-react";
+import { TokenForDocumentationT } from "../../../types/tokens";
+
+const ColorToken = ({ token }: { token: TokenForDocumentationT }) => {
+  switch (token.category) {
+    case "backgroundColor":
+      return (
+        <Box.New
+          borderColor="neutral-subtle"
+          borderRadius="full"
+          borderWidth="1"
+          style={{ backgroundColor: token.value }}
+          width="32px"
+          height="32px"
+        />
+      );
+    case "borderColor":
+      return (
+        <Box.New
+          borderRadius="full"
+          borderWidth="4"
+          width="32px"
+          height="32px"
+          style={{ borderColor: token.value }}
+        />
+      );
+    case "textColor": {
+      if (token.modifier === "icon") {
+        return (
+          <Box.New width="32px" height="32px">
+            <VStack as="div" align="center" justify="center" height="100%">
+              <PaletteIcon
+                title="a11y-title"
+                color={token.value}
+                fontSize="var(--ax-font-size-heading-medium)"
+              />
+            </VStack>
+          </Box.New>
+        );
+      }
+      return (
+        <Box.New width="32px" height="32px">
+          <VStack as="div" align="center" justify="center" height="100%">
+            <Heading size="medium" style={{ color: token.value }}>
+              Aa
+            </Heading>
+          </VStack>
+        </Box.New>
+      );
+    }
+    default:
+      // eslint-disable-next-line no-console
+      console.log("Unknown token category", token);
```

Comment:
Må vi alltid ha en default, eller kan denne fjernes?

Reply:
TODO: Fallback to "default" view

Reply:
Add the `vk-error` class, so it is picked up by Playwright test that checks for errors